From 0d98c57cd3276667655d3b23f8268f1249a9994d Mon Sep 17 00:00:00 2001
From: Derek Parker <deparker@redhat.com>
Date: Mon, 4 Feb 2019 01:49:43 -0800
Subject: [PATCH] Update doc location for RPM package

---
 pkg/terminal/command.go | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/pkg/terminal/command.go b/pkg/terminal/command.go
index 2927365447575b9a00418f72bdde87c5b58b9682..80d0df131613636c28800b5951f4bb5efba509f4 100644
--- a/pkg/terminal/command.go
+++ b/pkg/terminal/command.go
@@ -105,14 +105,14 @@ Type "help" followed by the name of a command for more information about it.`},
 
 	break [name] <linespec>
 
-See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/locspec.md for the syntax of linespec.
+See /usr/share/doc/delve/Documentation/cli/locspec.md for the syntax of linespec.
 
 See also: "help on", "help cond" and "help clear"`},
 		{aliases: []string{"trace", "t"}, cmdFn: tracepoint, helpMsg: `Set tracepoint.
 
 	trace [name] <linespec>
 
-A tracepoint is a breakpoint that does not stop the execution of the program, instead when the tracepoint is hit a notification is displayed. See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/locspec.md for the syntax of linespec.
+A tracepoint is a breakpoint that does not stop the execution of the program, instead when the tracepoint is hit a notification is displayed. See /usr/share/doc/delve/Documentation/cli/locspec.md for the syntax of linespec.
 
 See also: "help on", "help cond" and "help clear"`},
 		{aliases: []string{"restart", "r"}, cmdFn: restart, helpMsg: `Restart process.
@@ -182,7 +182,7 @@ Called with more arguments it will execute a command on the specified goroutine.
 
 	[goroutine <n>] [frame <m>] print <expression>
 
-See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/expr.md for a description of supported expressions.`},
+See /usr/share/doc/delve/Documentation/cli/expr.md for a description of supported expressions.`},
 		{aliases: []string{"whatis"}, cmdFn: whatisCommand, helpMsg: `Prints type of an expression.
 
 		whatis <expression>.`},
@@ -190,7 +190,7 @@ See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/expr.md for a des
 
 	[goroutine <n>] [frame <m>] set <variable> = <value>
 
-See $GOPATH/src/github.com/derekparker/delve/Documentation/cli/expr.md for a description of supported expressions. Only numerical variables and pointers can be changed.`},
+See /usr/share/doc/delve/Documentation/cli/expr.md for a description of supported expressions. Only numerical variables and pointers can be changed.`},
 		{aliases: []string{"sources"}, cmdFn: sources, helpMsg: `Print list of source files.
 
 	sources [<regex>]
-- 
2.16.5

